File: Lenguajes Orientados a Objetos/Actividad 7/Usuario Java/untitled/src/Usuario.java
public class Usuario {
    String nombre;
    String apellidos;
    int edad;


    void IniciarSesion()
    {
        System.out.println("El usuario " + this.nombre + " está iniciando sesión. ");
    }

    void cerrarSesion()
    {
        System.out.println("El usuario " + this.nombre + " está cerrando su sesión.");
    }

    void hacerReporte()
    {
        System.out.println("Reporte de usuario");
        System.out.println("Nombre completo: " + this.nombre + " " + this.apellidos);
        System.out.println("Edad: " + this.edad);

    }
}

File: Lenguajes Orientados a Objetos/Cosas de clase/Usuario.cs
// Mario Andrés Zaldívar de la Cruz
// #00586553

using System;

namespace User
{

class Usuario
{

public string Nombre {get; set;}
public string Apellidos {get; set;}
public int Edad {get; set;}

public Usuario(string nombre, string apellidos, int edad)
{
    Nombre = nombre;
    Apellidos = apellidos;
    Edad = edad;
    
}

void iniciarSesion()
{
    Console.WriteLine("El usuario " + this.nombre + " está iniciando sesión.");

}

void cerrarSesion()
{
    Console.WriteLine("El usuario " + this.nombre + " está cerrando sesión.");

}

public void hacerReporte()
{
    Console.WriteLine("Reporte de usuario");
    Console.WriteLine("Nombre completo: " + this.nombre + " " + this.apellidos);
    Console.WriteLine("Edad: " + this.edad);
}


}

    

}


